.bones/tasks/marky-n2mk.md

```
@@ -0,0 +1,105 @@
+status: closed
+type: task
+priority: 4
+owner: sethyanow@users.noreply.github.com
+---
+
+Phase 5 Step 2 of prepare-release skill uses literal PR_NUMBER in gh api command (line 371). Phase 5 runs in a separate session from Phase 3 where the PR was created, so the agent has no way to resolve the placeholder. Fix: add a gh pr list command to programmatically look up the release PR number, or document both options (extract from URL, gh CLI lookup).
+
+## Design
+
+## Goal
+
```

markdownlint-cli2 (0.21.0)

[warning] Trailing spaces
Expected: 0 or 2; Actual: 1

(MD009, no-trailing-spaces)